SubmitHGCalPGun.py

```diff
@@ -309,11 +313,13 @@ def submitHGCalProduction():
 
             cfgfile = basename +'.py'
             outfile = basename +'.root'
+            outdqmfile = basename.replace(opt.DTIER, 'DQM') +'.root'
```

Comment:
It'd be nice to have a template name that, if uploaded to a private DQM GUI, is silently digested correctly. This naming will miserably fail.

Reply:
Hi @rovere , Do you have any twiki with guidelines so that to have a properly name file?

Reply:
uhmm.
Here https://github.com/rovere/dqmgui/blob/index128/src/python/DQM/visDQMUtils.py
you can find how the name is inspected and decomposed. But I realize it's not immediate to understand.
Let me think about it a little longer and I can propose something.